docs/reference/commandline/system_prune.md

```diff
@@ -105,6 +105,7 @@ than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "b
 The currently supported filters are:
 
 * until (`<timestamp>`) - only remove containers, images, and networks created before given timestamp
+  * Cannot be used with `--volumes`
```

Comment:
Looks like this has a leading space; perhaps it's also better to either;

- add this to the "`until` ..." bullet to make it more clear which filter it applies to
- (possibly better) change this list to a table, and mention what objects each filter applies to